src/DevChatter.DevStreams.Infra.GraphQL/Types/ObjectGrapghType.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace DevChatter.DevStreams.Infra.GraphQL.Types
+{
+    public class ObjectGrapghType<T>
```

Comment:
Why does this exist? Looks like it might've been created by ReSharper when you made a typo and did an `Alt`+`Enter`, not noticing it was created.